core/includes/install.inc

```diff
@@ -114,7 +114,7 @@ function backdrop_get_schema_versions($module) {
     // Narrow this down to functions ending with an integer, since all
     // hook_update_N() functions end this way, and there are other
     // possible functions which match '_update_'. We use preg_grep() here
-    // instead of foreaching through all defined functions, since the loop
+    // instead of foreach-ing through all defined functions, since the loop
```

Comment:
I like word inventing as much as the next person 😅  ...but can't we just use "iterating through" here?

Reply:
Good point.